src/net/sourceforge/kolmafia/session/LimitMode.java

```diff
@@ -46,11 +57,28 @@ public void reset() {
     resetFunction().ifPresent(Runnable::run);
   }
 
+  public void finish() {
+    switch (this) {
```

Comment:
comment: llama forms are handled in `ChoiceControl`.

Reply:
Done